src/index.ts

```diff
@@ -60,7 +60,7 @@ app.all("*", async (context) => {
   // Make the request to the designated service
   const response = await fetch(url, {
     method: clone.method,
-    body: clone.body ? JSON.stringify(await clone.json()) : null,
```

Comment:
Wasn't there a reason you did this in the first place?

Reply:
Possibly not. Was thinking of 1c409a1271383aa636a3c06be0e2101a70a136c6

Reply:
Let me elaborate:

When a request reaches the worker, the body becomes a readable stream and is sent to the upstream service. However, our existing test cases expected a specific JSON format in the request body, causing them to fail when dealing with streams. To fix this, I initially modified the body handling by converting the stream to json, but later realised it wouldn't cover all scenarios like form data, json, etc. So, I decided to set the body as 'undefined' to allow flexibility and match with any type of data like streams, strings, or JSON.

We also have a test case [here](https://github.com/twodayslate/gateway/blob/181958307f9a8f9f80103b8dd0c4d03627b57e06/tests/index.test.ts#L356) that tests the GET request situation.